docs/sites.yml

```diff
@@ -10648,3 +10648,16 @@
   built_by: Chad Lee
   built_by_url: https://github.com/chadly
   featured: false
+- title: KAIGO in JAPAN
+  main_url: https://kaigo-in-japan.jp
+  url: https://kaigo-in-japan.jp
+  description: >
+    KAIGO in JAPAN is a website for those who are planning to work in the care work field in Japan. We built a multilingual site on Gatsby. One of them is a special language called Easy-Japanese with various ruby tags.
+  categories:
+    - Official
```

Comment:
```suggestion
```

We don't have a tag for `Official` in the site showcase, could you remove this line or change it to one like `Healthcare`? 🙂 

(you can see a lit of whitelisted tags here: https://github.com/gatsbyjs/gatsby/blob/master/docs/categories.yml)